src/SFA.DAS.EmployerApprenticeshipsService.Application/Queries/GetAccountTransferRole/GetAccountTransferRoleQuery.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using MediatR;
+using SFA.DAS.EAS.Application.Messages;
+
+namespace SFA.DAS.EAS.Application.Queries.GetAccountTransferRole
+{
+    public class GetAccountTransferRoleQuery : AuthorizedMessage, IAsyncRequest<GetAccountTransferRoleResponse>, IValidatableObject
+    {
+        public IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
```

Comment:
The `Validate` method shouldn't be needed here.

Reply:
Removed it